stripe-core/src/main/java/com/stripe/android/core/networking/RequestHeadersFactory.kt

```diff
@@ -164,5 +164,7 @@ sealed class RequestHeadersFactory {
         ) = "Stripe/v1 $sdkVersion"
 
         val CHARSET: String = Charsets.UTF_8.name()
+
+        const val UNDETERMINED_LANGUAGE = "und"
```

Comment:
unfortunately this is a private field in Locale.java, otherwise we could directly refer to it

Reply:
This change is required to pass the test for `Locale.ROOT` in `RequestHeadersFactoriesTest`